src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/ResourceActions/ResourceActions.ts

```diff
@@ -364,3 +370,20 @@ export var cancelActionDirective = (
         }
     };
 };
+
+export var goToMeetingActionDirective = (
+    adhTopLevelState : AdhTopLevelState.Service,
+    adhResourceUrl
+) => {
+    return {
+        restrict: "E",
+        transclude: true,
+        template: "<a class=\"{{class}}\" href=\"{{resourcePath | adhParentPath | adhResourceUrl:meeting}}\">" +
+            "<ng-transclude></ng-transclude> {{ 'TR__CANCEL' | translate }}</a>",
+        scope: {
+            resourcePath: "@",
+            class: "@",
+            meeting: "@"
+        }
+    };
+};
```

Comment:
This should definitely not be in core because meetings do not exist here as a concept.